Dockerfile

```diff
@@ -1,41 +1,28 @@
-FROM centos:6
+FROM ruby:2.3-slim-jessie
```

Comment:
Hmm. CentOS 6 image was originally chosen (I think, this was before my time) because it was closest to what we run on production. This new image is a different distro (we use Amazon Linux). There is actually an Amazon Linux Docker image that's available. Would prefer that if possible since it does change some of the dependency versions available like ImageMagic and ClamAV.

Reply:
Looks like the biggest issue is going to be with pdftk. There's no RPM published for it.

Reply:
From an engineering perspective the goal is to make working with docker similar to a native setup. Currently all the backend devs (including myself 😄) avoid docker because running specs etc is faster natively. If the Amazon Docker image can run as quickly (and can install pdftk) as the alpine/debian:jessie-slim breed of container then I'm all for it 👍

Reply:
@wyattwalter curious about LoE and risks migrating vets-api deployments to run on docker. 

Do we have enough testing in dev and staging to feel safe with our test suite never running on the deployment OS?

Reply:
I've been thinking about this ever since I looked at this PR. I think it's relatively low and we'd probably be ok to merge this in as-is with a TODO item to convert over to using it in production in the next month or so.

I looked at the way that we worked around the lack of pdftk rpm in production and it's as 🙈 as the old Dockerfile here so in my mind it's the difference in lift between using a Docker as a new packager or building out a bit of infra to build custom RPM's (the latter is not a thing that I want us to be doing).

Reply:
That being said, I don't believe that the switch to Alpine Linux by itself is what is creating the better performance here. The old Dockerfile has 10 image layers that got consolidated down to 1 just by getting rid of all the many yum transactions and other stuff into a single apt-get. That alone is going to have a HUGE impact.

Reply:
Thanks @wyattwalter - https://github.com/department-of-veterans-affairs/vets.gov-team/issues/10097.